setup.py

```diff
@@ -24,6 +24,10 @@
         'sonic_sfp',
         'sonic_thermal',
     ],
+    install_requires=[
+        'sonic-config-engine',  # For portconfig. TODO: Remove this dependency
+        'sonic-py-common'
```

Comment:
how do you name the python3 version? sonic-py3-common?

Reply:
do you need to capture it somewhere?

Reply:
The installed package has the exact same name. They reside in different Python directories (`/usr/local/lib/python2.X/dist-packages/sonic_py_common` and `/usr/local/lib/python3.X/dist-packages/sonic_py_common`. This will help ease the transition from Python 2 to Python 3.

The .whl files have `-py2` and -py3` suffices to differentiate.